src/gamebryo/gamebryosavegame.h

```diff
@@ -62,6 +62,12 @@ class GamebryoSaveGame : public MOBase::ISaveGame
     TYPE_WSTRING
   };
 
+  enum StringFormat
```

Comment:
Should be an enum class.

Reply:
This mirrors the StringType that's already in use, do you want me to change both?

Reply:
Yes that would be cleaner I think.